### PR TITLE
updated - working files

### PR DESCRIPTION
**Pull request by ibrahiminfinite**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,9 +1,10 @@
 <!DOCTYPE HTML>
 <html>
 <head>
-  <script src="jquery-2.1.3.min.js"></script>
-  <script src="node_modules/vis/dist/vis.js"></script>
-  <script src="matrix-vis.js"></script>
+<!-- <script src="jquery-2.1.3.min.js"></script>
+     <script src="node_modules/vis/dist/vis.js"></script>
+     <script src="matrix-vis.js"></script>-->
```
Comment (kegsay) [inline index.html]: Remove these comments, we don't need them.

File: matrix-vis.js [hunk 2]
```diff
@@ -13,7 +13,13 @@ var collapseEvents = true;
 var streamFrom = null;
 var scrollbackFrom = null;
 
-function init() {
+
+//new require()
```
Comment (kegsay) [inline matrix-vis.js]: Remove these comments, it's obvious.

File: matrix-vis.js [hunk 3]
```diff
@@ -13,7 +13,13 @@ var collapseEvents = true;
...
+var $= require("jquery");
+var vis= require("vis");
+//
```
Comment (kegsay) [inline matrix-vis.js]: Unnecessary.

**Comment by ibrahiminfinite** [comment]
closing this request, created new fork

**Pull request closed without merging**